CheckBox: update demo description (#2019)
DevExpress/devextreme-demos: JSDemos/Demos/CheckBox/Overview/description.md

> When you add the CheckBox component to an application, specify its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can have the following states: 

Reviewer: ```suggestion
When you add the CheckBox component to an application, specify the component's [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can be in the following states: 
```

> When you add the CheckBox component to an application, specify the component's [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can be in the following states: 

Reviewer 2: and here

- Checked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `true`).
- Unchecked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `false`).
- Indeterminate ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `null` or `undefined`).

> The component supports three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through this CheckBox states in the following order:

Reviewer: ```suggestion
The component supports the three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through the states of this CheckBox in the following order:
```

*Indeterminate → Checked → Unchecked → Indeterminate →  ...*

> Specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle the value change. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) of the "Handle value change" CheckBox is passed to the "Disabled" CheckBox. Click the first CheckBox to see how it affects the second one.

Reviewer: ```suggestion
Specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value changes. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) of the "Handle value change" CheckBox is passed to the "Disabled" CheckBox. Click the first CheckBox to see how it affects the second one.
```

> You can use the [iconSize](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#iconSize) property to specify a custom width and height for the CheckBox. To add a label to the CheckBox, specify the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) property.

Reviewer: ```suggestion
You can use the [iconSize](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#iconSize) property to specify custom dimensions for the CheckBox. To add a label to the CheckBox, specify the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) property.
```
